yomi.lua

```diff
@@ -326,8 +326,30 @@ local function yomi_check(task, content, digest, rule)
         elseif code == 401 or code == 403 then
           task:insert_result(true, 'YOMI_UNAUTHORIZED', 1, 'Unauthorized request returned ' .. code)
         elseif code == 202 then
-          if should_retransmit_hash() then
-            yomi_check_uncached()
+          local parser = ucl.parser()
+          local res, json_err = parser:parse_string(body)
+
+          if res then
+            local obj = parser:get_object()
+            local state = obj['state']
+
+            if state and state == 'WAITING' then
+              -- upload in progress
+              task:insert_result(true, 'YOMI_WAIT', 1, 'Upload in progress')
+              common.yield_result(task, rule, 'Upload in progress', 0.0, 'fail')
```

Comment:
Can we change with a more specific message?
```suggestion
              task:insert_result(true, 'YOMI_WAIT', 1, 'Hash analysis in progress')
              common.yield_result(task, rule, 'Hash analysis in progress', 0.0, 'fail')
```

Reply:
```suggestion
              task:insert_result(true, 'YOMI_WAIT', 1, 'File analysis in progress')
              common.yield_result(task, rule, 'File analysis in progress', 0.0, 'fail')
```